### PR TITLE
Issue #15: MERGE support added

### PR DESCRIPTION
**Pull request by Glitchfix**
https://github.com/RedisLabs/redisgraph-py/issues/15
Feature to merge node or relation added

USAGE:
```
tim = Node(label='person', alias=client.random_string(), properties={'name': 'Tim Doe', 'age': 25, 'gender': 'male', 'status': 'single'})
redis_graph.merge(tim)

tom = Node(label='person', properties={'name': 'Tom Doe', 'age': 23, 'gender': 'male', 'status': 'single'})
japan = Node(label='country', properties={'name': 'Japan'})
redis_graph.merge("""%s-[v:visited {purpose: "work"}]->%s"""%(tom,japan))
```

### CODE REVIEW

File: redisgraph/client.py [hunk 1]
```diff
@@ -159,3 +159,14 @@ def delete(self):
...
         """
         return self.redis_con.execute_command("GRAPH.DELETE", self.name)
+    
+    def merge(self, pattern):
+        """
+        Merge a node or relation.
+        """
+
+        query = 'MERGE '
+        if object:
```
Comment (swilly22) [inline redisgraph/client.py]: `if object:` I'm not sure I fully understand this condition, can you please explain this line?

**Comment by Glitchfix** [comment]
@swilly22  Can you please take a look at it and see if this is okay to merge?

**Comment by swilly22** [comment]
@Glitchfix Can you please address my question regarding `if object:`

**Comment by swilly22** [comment]
@Glitchfix thank you!

**Pull request merged**